scripts/parse_new_build_warnings.py

```diff
@@ -91,21 +94,28 @@ def RemoveFromSet(msg: str):
     RemoveFromSet(parsed)
     return new_build_warnings
 
-def append_warnings_to_file(output_file: str, warning_set: Set[str], old_build_name: str, new_build_name: str):
-    """Append the warnings to a given file_path"""
-    output_path = Path(output_file)
-    # Create possible missing output file directory
-    output_path.parent.mkdir(parents=True, exist_ok=True)
-
-    with open(output_path, 'a') as file:
-        introduction = f"# New Warnings from new build: {new_build_name} old build: {old_build_name}\n\n"
-        file.write(introduction)
-        for warning in warning_set:
-            file.write(warning)
-        # Add extra new lines for better visual
-        file.write("\n\n")
-
-def parse_new_build_warnings_from_directory(old_build_directory: str, new_build_directory: str, output: str):
+def parse_target(file_name: str)-> str:
+    """Parse only the target info from the post commit build log formatted file name"""
+    parts = file_name.split('-')
+    # Replace the hash with a wildcard regex
+    HASH_INDEX = 3
+    del parts[HASH_INDEX]
+    BUILD_LOG_INDEX=-3
+    return '-'.join(parts[:BUILD_LOG_INDEX])
```

Comment:
Can you add a comment what it returns? I think it returns `mode-arch-abi-multilib`. I also don't see where the wildcard regex comes in

Reply:
from what it looks like, it's `hashless = parse_target(file_name)` so maybe remove the comment? I might be missing something though

Reply:
Lin e 100 comment seems to be leftout after some modifications to the commit. I'll remove that comment

Reply:
Also I'll add additional description about the convesion. I'll specify that `mode-arch-abi-hash-multilib-build-log-stderr.log` will be converted to `mode-arch-abi-multilib-build-log-stderr.log`